src/main/java/com/revature/g2g/models/SurveySkillDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.revature.g2g.models;
```

Comment:
Is this dto used anywhere?

Reply:
Survey DTO still makes use of it, I'll be happy to remove it entirely if there's a better way.

Reply:
Found out that nothing calls that constructor anymore, axing the constructor and SurveySkillDTO.